com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -517,16 +518,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed an issue with refraction model and ray traced recursive rendering (case 1198578).
 - Fixed an issue where a dynamic sky changing any frame may not update the ambient probe.
 - Fixed cubemap thumbnail generation at project load time.
-- Fixed cubemap thumbnail generation at project load time. 
+- Fixed cubemap thumbnail generation at project load time.
```

Comment:
Are these chagnes (and below) nececarry keep the diff as small as possible it seems like a whitespace/newline issue?

Reply:
From what I can tell these sneaked in when resolving a merge conflict using Github Desktop (which used Atom).
Maybe we want to keep these whitespace changes for consistency ? The file itself does need to be touched by this PR.

Reply:
Update: Whitespace changes removed.